server/util/log/gcp/gcp.go

```diff
@@ -0,0 +1,47 @@
+package gcp
+
+import (
+	"context"
+
+	"cloud.google.com/go/logging"
+	"github.com/rs/zerolog"
+)
+
+func NewLogger(projectID string, logID string) (zerolog.LevelWriter, error) {
+	client, err := logging.NewClient(context.Background(), projectID)
+	if err != nil {
+		return nil, err
+	}
+	return &logWriter{ctx: context.Background(), logger: client.Logger(logID)}, nil
+}
+
+type logWriter struct {
+	ctx    context.Context
+	logger *logging.Logger
+}
+
+func (l *logWriter) Write(p []byte) (int, error) {
+	l.logger.StandardLogger(logging.Default).Print(string(p))
+	return len(p), nil
+}
+
+func (l *logWriter) WriteLevel(level zerolog.Level, p []byte) (int, error) {
```

Comment:
I haven't seen what these logs look like in the gcp viewer yet, but wondering if we manually need to add the same structured-logging stuff that zerolog is adding and/or the keys from the context (like in enrichEventFromContext)

Reply:
Fixed this: 
![image](https://user-images.githubusercontent.com/735684/203181025-9be8f9df-1532-4bd0-91d0-760b578acaef.png)

Reply:
:heart_eyes: